src/plugins/core/plugin_minify_html_literals.ts

```diff
@@ -0,0 +1,33 @@
+import { minifyHTMLLiterals } from 'minify-html-literals';
+import { Context } from '../../core/Context';
+
+export function pluginMinifyHtmlLiterals(...options: any) {
```

Comment:
can you add an interface IPluginMinifyHtmlLiterals to follow the conventions?

Reply:
yes